GuiApp/main.cpp

```diff
@@ -57,7 +57,10 @@ int main(int argc, char *argv[])
 
     auto args = app.arguments();
     QString fileToLoad = "";
-    if (args.length() == 2)
+    if(app.fileToLoad.length() > 0) {

```

Comment:
app.fileToLoad defaults to "", so this could be simplified to
fileToLoad = app.fileToLoad
if( args.length() == 2)
{...